engines/myst3/detection.cpp

```diff
@@ -228,6 +267,27 @@ class Myst3MetaEngineDetection : public AdvancedMetaEngineDetection<Myst3GameDes
 	const DebugChannelDef *getDebugChannels() const override {
 		return debugFlagList;
 	}
+
+	DetectedGame toDetectedGame(const ADDetectedGame &adGame, ADDetectedGameExtraInfo *extraInfo) const {
+		DetectedGame game = AdvancedMetaEngineDetection::toDetectedGame(adGame);
+
+		// The AdvancedDetector model only allows specifying a single supported
+		// game language. The 10th anniversary edition Myst III is multilanguage.
+		// Here we amend the detected games to set the list of supported languages.
+		if ((reinterpret_cast<const Myst3::Myst3GameDescription *>(
+				adGame.desc)->flags & 0xff00) != Myst3::kLayoutFlattened) {
```

Comment:
What is the 0xff00 value for the flags?

Reply:
OK... you got me. I was a bit lazy to introduce a constant.
This is now done. :wink: